docs/reference/experimental/mixins/base_json_schema.md

```diff
@@ -0,0 +1,3 @@
+# BaseJSONSchema
+
+::: synapseclient.models.mixins.BaseJSONSchema
```

Comment:
The other dataclasses that can be returned or used in any of these models or mixins should also be added to this markdown page. For example:

`::: synapseclient.models.mixins.JSONSchemaBinding`


The consequence of not including this is that you cannot click on the return type when you look at the function in the API docs, and it's unclear from the doc page what fields will be available on that object:

![image](https://github.com/user-attachments/assets/07a84992-3437-4dc0-a480-c0aa42b39740)